Changelog.md

```diff
@@ -19,6 +19,7 @@ Compiler Features:
  * Standard JSON: New option ``modelCheckerSettings.targets`` allows specifying which targets should be checked. The valid options are ``all``, ``constantCondition``,
    ``underflow``, ``overflow``, ``divByZero``, ``balance``, ``assert``, ``popEmptyArray``, where the default is ``all``. Multiple targets can be chosen at the same time,
    separated by a comma without spaces: ``underflow,overflow,assert``.
+ * Code Generator: Implemented a shortcut  for ``<address>.code.length`` that generates ``extcodesize(<address>)``.
```

Comment:
Maybe better to write `* Code Generator: Add support for <address>.code.length that generates extcodesize(<address>)` or `* Code Generator: Implemented the shortcut <address>.code.length that generates extcodesize(<address>).`, because `<address>.code.length` itself is the shortcut, right?

Reply:
No, `.code.length` existed before but it is cheaper now. What about `reduce cost of <address>.code.length by uisng extocdesize directly`?

Reply:
Ah great, finally I understood now why the yul stuff was already working :)